tla-io/src/main/scala/at/forsyte/apalache/io/lir/PrettyWriter.scala

```diff
@@ -41,12 +43,42 @@ class PrettyWriter(
   private def prettyWriteDoc(doc: Doc): Unit = writer.write(pretty(doc, layout.textWidth).layout)
 
   def write(mod: TlaModule, extendedModuleNames: List[String] = List.empty): Unit =
-    prettyWriteDoc(toDoc(mod, extendedModuleNames))
+    prettyWriteDoc(modToDoc(mod, extendedModuleNames))
+
+  /**
+   * Pretty-prints the given decl twice: Once as valid TLA, and once (before the TLA expression) in a comment, where
+   * names of NameExs are replaced by their mapping in the NameReplacementMap. The output looks, for example, like this:
+   *
```

Comment:
Could you include an example of the `NameReplacementMap` that produces the example below?